couchbase/kv/v1/kv.proto

```diff
@@ -78,10 +87,14 @@ message GetAndTouchRequest {
     google.protobuf.Timestamp expiry_time = 5;
     uint32 expiry_secs = 6;
   }
+  optional CompressionEnabled compression = 7;
```

Comment:
Can I safely assume the default behaviour (if compression is not sent) is to disable compression.  E.g. we won't ever get `content_compressed` until we explicitly opt into it?

Reply:
Correct.